src/js/util/shabad/get-audio-url.ts

```diff
@@ -0,0 +1,58 @@
+import axios from "axios";
+import { API_URL, S3_BUCKET_URL } from "@/constants";
+
+export const checkAPI = async function () {
+  let uri = `${API_URL}health/`;
+  let user = 'admin';
+  let password = process.env.AUDIO_API_PASS;
+  let APIisHealthy = false;
+
+  const res = await axios.get(uri, {
+    headers: {
+      'Accept': '*/*',
+      'Authorization': `Basic ${btoa(user + ':' + password)}`,
+      'Content-Type': 'application/json',
+      'Connection': 'keep-alive'
+    }
+  }).then((response) => {
+    if (response.data.ok) {
+      APIisHealthy = true;
+    }
+  }).catch((error) => {
+    console.error({
+        message: "Audio API is down!",
+        error
+    })
+    return {data:{status: "error"}}
+  });;
+
+  return APIisHealthy;
+}
+
+export const getAudio = async function (info: any) {
```

Comment:
@MySmilingTurban this function is returning both a boolean and/or shabadURL. 
`return hasAudio  or `shbdUrl`. This is like violating the single responsibility principle here. Can you make sure this function do only 1 part and should rename this function accordingly